modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -196,8 +268,102 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
       enterprise: enterprise,
     };
 
-    const keychains = await this.baseCoin.keychains().add(createBitGoMPCParams);
-    return keychains;
+    return await this.baseCoin.keychains().add(createBitGoMPCParams);
+  }
+
+  async getBackupEncryptedNShare(
+    backupShare: BackupKeyShare,
+    recipientIndex: number,
+    recipientGpgPublicArmor: string,
+    isThirdPartyBackup = false
+  ): Promise<EncryptedNShare> {
+    let backupToBitgoShare: EncryptedNShare;
+    if (isThirdPartyBackup && backupShare.bitGoHeldKeyShares?.keyShares) {
+      const backupToOtherApiShare = backupShare.bitGoHeldKeyShares.keyShares.find(
+        (keyShare) => keyShare.from === 'backup' && keyShare.to === getParticipantFromIndex(recipientIndex)
+      );
+      if (!backupToOtherApiShare) {
+        throw new Error('Missing Backup to Bitgo key share');
+      }
+      backupToBitgoShare = await encryptNShareFromAPIKeyShare(backupToOtherApiShare, recipientGpgPublicArmor);
```

Comment:
If we get the `backupToBitgoShare` already from BitGo then it should already be encrypted and we shouldn't need to do that in the SDK. In that case I'm expecting we pass it back to BitGo just the way we got it from BitGo.

Reply:
Yup you're right. I will remove that encryption